jdbc/src/test/java/org/apache/iotdb/jdbc/IoTDBResultMetadataTest.java

```diff
@@ -120,7 +124,7 @@ public void testGetColumnType() throws SQLException {
     } catch (Exception e) {
       flag = true;
     }
-    assertEquals(flag, true);
+    assertTrue(flag);
```

Comment:
If you also return 0 when it's null, like I suggested above, don't forget to change assert here.

Reply:
fixed